tests/ssl-opt.sh

```diff
@@ -5575,25 +5573,25 @@ run_test    "DTLS fragmenting: 3d, gnutls server, DTLS 1.0" \
 ##             0 \
 ##             -c "fragmenting handshake message" \
 ##             -C "error"
-
-requires_openssl_legacy
-requires_config_enabled MBEDTLS_SSL_PROTO_DTLS
-requires_config_enabled MBEDTLS_RSA_C
-requires_config_enabled MBEDTLS_ECDSA_C
-requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_1
-client_needs_more_time 4
-run_test    "DTLS fragmenting: 3d, openssl server, DTLS 1.0" \
-            -p "$P_PXY drop=8 delay=8 duplicate=8" \
-            "$O_LEGACY_SRV -dtls1 -verify 10" \
-            "$P_CLI dtls=1 debug_level=2 \
-             crt_file=data_files/server8_int-ca2.crt \
-             key_file=data_files/server8.key \
-             hs_timeout=250-60000 mtu=512 force_version=dtls1" \
-            0 \
-            -c "fragmenting handshake message" \
-            -C "error"
-
-## see comment on the previous-previous test
+##
+## requires_config_enabled MBEDTLS_SSL_PROTO_DTLS
```

Comment:
Rather than comment out tests which just hides them, we should be using `skip_next_test` which forces them to be always skipped.

That was they'll always be listed in the stats as a test which is present, but wasn't executed.

Reply:
I agree, but I was trying to keep this PR minimal in the hope that it would make it into the release. I think @hanno-arm has a PR that addresses previous feedback, probably including this on (skip the tests rather than comment them). If that's the case, I suggest closing this PR and doing it properly in Hanno's PR instead.

Reply:
Oops, I was referring to #1988 but that has already been merged, so I'll update this PR instead.